Resources/Maps/fland.yml

```diff
@@ -180555,11 +180555,15 @@ entities:
       parent: 13329
   - uid: 11854
     components:
-    - type: MetaData
-      name: Mailroom Garage Door Opener
     - type: Transform
-      pos: 21.53215,-13.842299
+      pos: 21.52355,-14.18487
```

Comment:
```suggestion
      pos: 21.53215,-13.842299
```

Reply:
Remote is still on same initial table, just with slightly different positioning.